src/commands/xkcd.ts

```diff
@@ -0,0 +1,144 @@
+import { ApplicationCommandOptionType, EmbedBuilder } from 'discord.js';
+import axios from 'axios';
+
+interface GetComicResponse {
+	month: string;
+	num: number;
+	link: string;
+	news: string;
+	safe_title: string;
+	transcript: string;
+	alt: string;
+	year: string;
+	title: string;
+	day: string;
+	img: string;
+}
+
+// a quick call to see the latest comic.
+// @return the data of the most recent comic, or an empty response with num set to -1 to indicate an error.
+async function _latestCheck(): Promise<GetComicResponse> {
+	try {
+		const { data, status } = await axios.get<GetComicResponse>('https://xkcd.now.sh/?comic=latest');
+		if (status !== 200) throw new Error(`${status}`);
+		return data;
+	} catch (error_) {
+		console.log(error_);
+		const error: GetComicResponse = {
+			month: '',
+			num: -1,
+			link: '',
+			news: '',
+			safe_title: '',
+			transcript: '',
+			alt: '',
+			year: '',
+			title: '',
+			day: '',
+			img: '',
+		};
+		return error;
+	}
+}
+
+export const xkcd: GlobalCommand = {
+	name: 'xkcd',
+	description: 'Fetches the most recent xkcd comic, or a selected one.',
+	requiresGuild: false,
+	options: [
+		{
+			name: 'number',
+			description: 'The index number of the comic you would like to have',
+			type: ApplicationCommandOptionType.Integer,
+		},
+	],
+
+	// entry point for command execution
+	async execute({ options, reply, sendTyping }) {
+		// sends an ephemeral message to the user
+		// @param reply: the reply method given in the execute method's CommandContext
+		// @param content: the string that should be sent to the client to let them know the error.
```

Comment:
This looks like you're going for JSDoc, but `//` comments don't get parsed that way. You might be looking for `/** */`:
```suggestion
		/**
		 * Sends an ephemeral message to the user.
		 * @param content The string that should be sent to the client to let them know the error.
		 */
```
This gets parsed in VS Code as follows:

![Screen Shot 2022-10-01 at 18 48 49](https://user-images.githubusercontent.com/1500092/193433327-35c094ea-a398-4755-95fd-e3fa64c84bfa.png)

Reply:
Done for both functions.